src/waveform/visualplayposition.cpp

```diff
@@ -7,6 +7,13 @@
 #include "util/math.h"
 #include "waveform/vsyncthread.h"
 
+namespace {
+constexpr int kMaxOffsetRange = 2; // The offset is limited to two callback sizes
```

Comment:
It's not obvious what unit of measurement applies here and why this constant is multiplied by 1000 at the two places where it is actually used.